draft-ietf-quic-transport.md

```diff
@@ -2070,6 +2150,11 @@ Connection ID:
 
 : A 64-bit connection ID.
 
+Stateless Reset Token:
+
+: A 128-bit value that will be used to for a stateless reset when the associated
+  connection ID is used (see {{stateless-reset}}).
```

Comment:
Yeah, open to suggestions for how this might be improved.  Nothing I tried seemed to work that well.